README.md

```diff
@@ -6,9 +12,18 @@ docker-credential-helpers is a suite of programs to use native stores to keep Do
 
 Go to the [Releases](https://github.com/docker/docker-credential-helpers/releases) page and download the binary that works better for you. Put that binary in your `$PATH`, so Docker can find it.
 
-### Building from scratch
+## Building
+
+You can easily build the credential helpers using Docker:
+
+```shell
+# create builder
+$ docker buildx create --use
+# build credential helpers and output to ./bin/build
+$ docker buildx bake "https://github.com/docker/docker-credential-helpers.git"
+```
 
-The programs in this repository are written with the Go programming language. These instructions assume that you have previous knowledge about the language and you have it installed in your machine.
+Or if the toolchain is already installed on your machine:
 
 1 - Download the source and put it in your `$GOPATH` with `go get`.
```

Comment:
Looks like this section is in need of a rewrite as well (`go get` won't work anymore to clone the repository)

Reply:
Yes if we bump to Go 1.19 this has to be changed

Reply:
I prepared https://github.com/docker/docker-credential-helpers/pull/226 for go 1.18 (but best to do after this one)